Book-A-desk.Domain.Tests/Reservation.Validator.Tests.fs

```diff
@@ -79,12 +113,12 @@ let ``GIVEN A Book-A-Desk Reservation command WITH a date greater than today, WH
 let ``GIVEN A Book-A-Desk Reservation command WITH an invalid office id, WHEN validating, THEN validation should fail`` () =
     let commandWithInvalidOfficeId =
         {
-            EmailAddress = EmailAddress "anEmailAddress@fake.com"
+            EmailAddress = EmailAddress $"email@{domainName}"
```

Comment:
Not for this PR:
An idea: the validation for the email address could be moved in the EmailAddress constructor so it's impossible to have an invalid email in the domain